modules/compliance-rescan.adoc

```diff
@@ -10,3 +10,8 @@ Typically you will want to re-run a scan on a defined schedule, like every Monda
 ----
 $ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
 ----
+
+[NOTE]
+====
+When the scan setting `default-auto-apply` label is applied, the annotation `compliance.openshift.io/rescan=` triggers a cluster reboot.
```

Comment:
uhm... maybe it would be better to expand more on the issue.

`default-auto-apply` will apply remediations automatically and will auto update outdated remediations too.

If there are remediations that haven't been applied due to dependencies, or remediations that had been outdated, re-scanning will make them apply and that **might** trigger a reboot. Note that only remediations that use MachineConfig objects trigger reboots.

What Xiaojie noticed was that when she would scan, the remediations would apply, and the second re-scan would reboot the nodes again; this was because the moderate profile specifically has remediations that have dependencies. So on the second scan the dependencies had been met, which led to the reboot.

If there are no updates nor dependencies to be applied, no reboot will happen.

Reply:
correct, usually the rescan wasn't trigger a reboot.
It only happens when there are some complianceremediations in MissingDependencies status.
If there are no updates nor dependencies to be applied, no reboot will happen.
$ oc get complianceremediations | grep MissingDependencies
rhcos4-moderate-master-configure-usbguard-auditbackend                                              MissingDependencies
rhcos4-moderate-master-service-usbguard-enabled                                                     MissingDependencies
rhcos4-moderate-master-usbguard-allow-hid-and-hub                                                   MissingDependencies
rhcos4-moderate-worker-configure-usbguard-auditbackend                                              MissingDependencies
rhcos4-moderate-worker-service-usbguard-enabled                                                     MissingDependencies
rhcos4-moderate-worker-usbguard-allow-hid-and-hub                                                   MissingDependencies

Reply:
@JAORMX @xiaojiey Thank you! I made an update. PTAL. Thanks!